.travis.yml

```diff
@@ -7,16 +7,23 @@ cache:
   yarn: true
   directories:
   - node_modules
-before_install:
-- echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" >> ~/.npmrc
-before_script:
-- yarn install
-- yarn build
 script:
 - yarn lint
+- yarn lerna bootstrap
```

Comment:
can you remove this line and add a postinstall command like https://github.com/cozy/cozy-libs/blob/master/package.json#L16

`"postinstall": "lerna bootstrap"`